scss/_patterns_code-snippet.scss

```diff
@@ -44,6 +45,15 @@ $color-snippet-heading-bg: rgba($color-x-dark, 0.08);
       }
     }
 
+    .p-code-snippet__block--numbered {
+      counter-reset: line-numbering;
+      padding: $sph-inner--small 0;
```

Comment:
Same as with legacy component - does it need to override the padding? I guess we can accomodate the padding on the lines with padding on the block, and avoid overriding this.